tests/functional/testplan/testing/multitest/test_multitest_drivers.py

```diff
@@ -18,6 +22,7 @@
 
 
 CUSTOM_RUNPATH = os.path.join(tempfile.gettempdir(),
```

Comment:
You can use the `runpath` shared fixture to get a temporary runpath for tests that gets automatically cleaned up.